index.html

```diff
@@ -800,7 +800,9 @@ <h3>Credentials</h3>
 <a>verifiable credential</a>, but abstracts the details about how <a>claims</a>
 are organized into information <a>graphs</a>, which are then organized into
 <a>verifiable credentials</a>. <a href="#info-graph-vc"></a> below shows a
-more complete depiction of a <a>verifiable credential</a>, which is normally
+more complete depiction of a <a>verifiable credential</a>
+secured with Data Integrity Proofs [[VC-DATA-INTEGRITY]].
+<a>Verifiable credentials</a> secured in this manner are
```

Comment:
I do not know yet how this PR will be reconciled with #1389 (there are many overlapping issues). While I _agree_ with the change proposal here, I would request not to adopt it ***for now***. The reason is that I plan a separate PR, following the discussion in #1135, which would also add a JWT-based diagram alongside the current one, and which, therefore, makes a lot of changes on this text. The updated text (which is already on my machine) will (already does) include this type of remark.

I plan to submit the related PR once the current, pre-CR PR-s are finalized (merged or closed).

Thx.